segway/gmtk/params.py

```diff
@@ -200,22 +253,52 @@ def get_header_info(self) -> str:
         return " ".join(line)
 
 
-class Mean(OneLineKind):
+class Mean(OneLineArray):
     """
     A single Mean object.
     __init__ and __str__ methods defined in superclass.
     """
     kind = OBJ_KIND_MEAN
 
 
-class Covar(OneLineKind):
+class Covar(OneLineArray):
     """
     A single Covar object.
     __init__ and __str__ methods defined in superclass.
     """
     kind = OBJ_KIND_COVAR
 
 
+class VirtualEvidence:
+    """
+    A Virtual Evidence object, with syntax inferred from the input.master
+    template implementation.
```

Comment:
in Segway?

Reply:
Unfortunately yes. I found no documentation on how this object is expected to behave in general, so the syntax is inferred from the template file under the previous Segway implementation. Let me know if I should remove the comment on the source.